avm/res/document-db/database-account/sql-database/container/main.bicep

```diff
@@ -0,0 +1,95 @@
+metadata name = 'DocumentDB Database Account SQL Database Containers'
+metadata description = 'This module deploys a SQL Database Container in a CosmosDB Account.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Conditional. The name of the parent Database Account. Required if the template is used in a standalone deployment.')
+param databaseAccountName string
+
+@description('Conditional. The name of the parent SQL Database. Required if the template is used in a standalone deployment.')
+param sqlDatabaseName string
+
+@description('Required. Name of the container.')
+param name string
+
+@description('Optional. Indicates how long data should be retained in the analytical store, for a container. Analytical store is enabled when ATTL is set with a value other than 0. If the value is set to -1, the analytical store retains all historical data, irrespective of the retention of the data in the transactional store.')
+param analyticalStorageTtl int = 0
+
+@description('Optional. The conflict resolution policy for the container. Conflicts and conflict resolution policies are applicable if the Azure Cosmos DB account is configured with multiple write regions.')
+param conflictResolutionPolicy object = {}
+
+@maxValue(2147483647)
+@minValue(-1)
+@description('Optional. Default time to live (in seconds). With Time to Live or TTL, Azure Cosmos DB provides the ability to delete items automatically from a container after a certain time period. If the value is set to "-1", it is equal to infinity, and items dont expire by default.')
+param defaultTtl int = -1
```

Comment:
Please make nullable

Reply:
As discussed on Teams, the "-1" here is not just a workaround, but a documented value, so I suggest to keep it.
See: https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/time-to-live